site/content/integrate/plugins/interactive-dialogs/_index.md

```
@@ -621,6 +622,48 @@ The `datetime_config` object groups date/datetime configuration into a single ne
 - `"time_interval": 60` creates options: 00:00, 01:00, 02:00, 03:00, etc.
 - Invalid: `"time_interval": 7` (7 is not a divisor of 1440)
 
+### Collapsible elements
+##### Minimum Server Version: 11.10
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)